app/policies/vaos_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Should this live here, or within the module (like how you created a `services` directory in the module instead of using the one in the main app)?